functions/v2/imagemagick/README.md

```diff
@@ -0,0 +1,55 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Functions ImageMagick sample
+
+This sample shows you how to blur an image using ImageMagick in a
+Storage-triggered Cloud Function.
+
+View the [source code][code].
+
+[code]: main.py
+
+## Deploy and Test
+
+1. Follow the [Cloud Functions quickstart guide][quickstart] to setup Cloud
+Functions for your project.
+
+1. Clone this repository:
+
+        git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+        cd python-docs-samples/functions/imagemagick
```

Comment:
I think we want to avoid including v2 in the directions, but eventually there should be a strategy to make v2 be what folks get to by default whether it's a symlink or moving what exists now at the top level into v1 and what exists in v2 moved into the base.

Reply:
These directions are specific to this directory.

(IMO, we should update the directions if/when we move the sample itself.)